inlong-manager/manager-web/src/main/resources/application-dev.properties

```diff
@@ -82,3 +82,7 @@ data.cleansing.interval.seconds=1800
 data.cleansing.before.days=10
 # The maximum size of data to be deleted in batch, default is 100
 data.cleansing.batchSize=100
+
+# If turned on, in configuring group status will be set to failed after restart
+# Only set this to true when a single manager instance is deployed
+group.status.cleanup.enabled = false
```

Comment:
It is recommended to add an interface to change the group status from "in configuration" to "configuration failure" (for modify_time is x hours ago),

so as to avoid the unavailability of this configuration when multiple inlong manager applications are used,

and also more flexibly support users to call and process on demand.

Reply:
OK. Turns out the reset interface already exists. Then this task is not needed.